docs/codehealth_rotation.md

```diff
@@ -0,0 +1,87 @@
+# Code Health
+
+## Overview
+
+This is an outline for the CodeHealth rotation, which aims to proactively manage
+technical debt and maintain our codebase by keeping it up-to-date with upstream
+Chromium. These tasks are essential for keeping the codebase current, clean, and
+maintainable.
+
+Additionally, CodeHealth work is a great opportunity to familiarise oneself with
+different areas of the codebase, modernising the code, introducing new language
+features, and applying the style guide's best practices.
+
+## Rotation Process
+
+To ensure steady, incremental improvements, a weekly rotation is in place.
+
+  * **Staff**: Each `brave-core` pod provides a different volunteer to take up
+    on a CodeHealth task each week.
+  * **Cycles**: The expected time commitment is up to 4 hours per volunteer
+    (per week).
+  * **Scope**: Work should be kept small enough to be manageable and to not
```

Comment:
What if folks on my team are already working on major refactoring with the explicit goal of paying down tech debt? These refactorings might not be small. Is that under scope too?

Reply:
Yeah, you have a point, maybe I should add a paragraph that clarifies that teams are still incentivised to have their own refactoring routines unrelated to the CodeHealth rotation, otherwise we could end up with thing pawned over our backlog.

Reply:
I've added a whole new section to explain that CodeHealth doesn't mean that teams just don't have their own refactoring routines. Thanks for pointing that out.